colortv/docs/androidTVSDK.md

```diff
@@ -591,83 +591,214 @@ public class MainActivity extends Activity {
+<br /><br />
+
+
+#### **ID:** ctv_autoPlayTimerContainer
+* **View type:** `FrameLayout`
+* **Description:** Used to inject countdown timer to automatically start playing the first recommended video
+* **Animation:** Hides on any interaction on items list or when timer reaches 0.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_videoPreviewContainer
+* **View type:** Any view extending the ViewGroup class (eg. `LinearLayout`, `RelativeLayout`)
+* **Description:** Used to inject our `VideoPlayer` for previews playback. Check our [custom layout](https://github.com/color-tv/android-SampleApp/blob/master/SampleApp/app/src/main/res/layout/custom_item_layout_2.xml) if you would like to oversize the preview video.
+* **Animation:** Plays the preview and hides the thumbnail on focus (only if the preview is available) and stops playing the preview and shows the thumbnail on unfocus.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_ivVideoThumbnail
+* **View type:** `ImageView`
+* **Description:** Used to display the video thumbnail.
+* **Animation:** Hides on focus if there is preview available and shows on unfocus.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_ivAppLogo
+* **View type:** `ImageView`
+* **Description:** Used to display logo.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_ivPlayButton
+* **View type:** `ImageView`
+* **Animation:** Hides on focus and shows on unfocus.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_ivBlackMask
+* **View type:** `ImageView`
+* **Animation:** Hides on focus and shows on unfocus.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_tvTitle
+* **View type:** `TextView`
+* **Description:** Displays the video title.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_tvDescription
+* **View type:** `TextView`
+* **Description:** Displays the video description.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_tvDuration
+* **View type:** `TextView`
+* **Description:** Displays the video duration.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_tvGenre
+* **View type:** `TextView`
+* **Description:** Defines how each genre should look like. Visibility must be set to `Visibility.GONE`.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_llGenres
+* **View type:** `LinearLayout`
+* **Description:** Used to contain genres, which will look like the `TextView` with id `ctv_tvGenre`.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_ivFavoriteIcon
+* **View type:** `ImageView`
+* **Description:** Shows `ctv_ivFavoriteContainer` on click.
+* **Device:** ALL
+
+</details>
+
+Check our example [item layouts](https://github.com/color-tv/android-SampleApp/tree/master/SampleApp/app/src/main/res/layout) for better a understanding of the customization options.
 
 #### setRowCount(Device device, int rowCount)
 
-This method is used to set the number of rows for specified device type.
+This method is used to set the number of rows for a specified device type.
```

Comment:
I'd change it to:  ...number of rows in RecyclerView with ctv_rvGrid id for specified device type.